### PR TITLE
Saml

### PR DESCRIPTION
**Pull request by terra4383**
Thanks for the input and clarification today.  Please let me know if you think any additional edits are necessary.

### CODE REVIEW

**Comment by terra4383** [comment]
@thedunster @jday3 @jleo32 @SteveFeldman Can I get at least one approval before merging this in on the Docs side? (The article won't be linked up or shown on the site until we're ready for next release of EOP).

**Comment by SteveFeldman** [comment]
We really need @thedunster to put together a Troubleshooting document as well. We should have something that helps customers troubleshoot a bad configuration.

File: content/admin/teamserver/config/integration/SSO.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+## EOP
+
+For Enterprise-On-Premises customers, Single Sign-On is configured at the System level by the SuperAdmin.
+
+### Notes About Switching to SSO
+
+If a System Admin decides to set up SSO, it is important to note the following:
+
+Contrast does not support user provisioning, therefore all users must have an existing account in TeamServer before switching the authentication method. Additionally, if users are identified with a user ID rather than an email address, those accounts won’t automatically transfer over to the SSO configuration and will need to be recreated.
```
Comment (thedunster) [inline content/admin/teamserver/config/integration/SSO.md]: > therefore all users must have an existing account in TeamServer before switching the authentication method. 

Maybe something more like "an existing account or one created in TeamServer before..." Accounts can be created with SSO auth mode.

**Comment by terra4383** [comment]
I removed mention of LDAP config on SaaS. @thedunster let me know if you need to discuss updating the document or if you want a ticket for the Troubleshooting/SAML 2.0 spec.

**Comment by thedunster** [comment]
Add a little section about trouble shooting with the IDP's metadata.

**Pull request merged**